### PR TITLE
Many updates

### PR DESCRIPTION
**Pull request by jhdalek55**
I am doing this as a pull request, rather than committing it directly because I ended up making more changes than anticipated. Also, I don't have the localhost set-up on my computer here at home, so I couldn't check for weird glitches. However, I was super-careful as I was making these changes, so I think everything is OK. I just wanted eyes on it before it was pushed to master.

In this PR, I:

- Added five photos for the new interns/undergrads working with us now. I have at least two more to add, but I'm waiting on photos and information.

- Added bio info for the new group and made sure to add them to their respective projects
_Added abstracts to three papers, and links to two
- Added Ghada's Microcash paper, with abstract and links
- Filled in some missing publication information on people pages
- Removed references to the Google doc versions of our Uptane implementation and added reference and link to the Uptane Standard.

Please look over when you can. When you merge, remember you have to push it to master for me.

### CODE REVIEW

File: _data/data.yml [hunk 1]
```diff
@@ -1013,37 +1086,55 @@ publications:
...
+
     - &moore_crashsim_issre_19
       anchor: moore_crashsim_issre_19
       title: "Charting a Course Through Uncertain Environments: SEA Uses Past Problems to Avoid Future Failures"
       authors:
-        - name: "P. Moore, J. Cappos, P. Frankl,  and T. Wies"
+        - name: "P. Moore, J. Cappos, P. Frankl, and T. Wies"
       project: *crashsimulator
-      booktitle: "to be presented at the 30th IEEE International Symposium on Software Reliability Engineering (ISSRE'19)"
+      booktitle: "30th IEEE International Symposium on Software Reliability Engineering (ISSRE'19) <br/> Best Paper Award"
```
Comment (JustinCappos) [inline _data/data.yml:1109]: Is using angle brackets allowed in yml files?

Comment (jhdalek55) [reply _data/data.yml:1109]: To be honest, I'm not sure. I noticed this was how the "best paper" was highlighted in Dan's paper. I can remove it. Is this why checks failed.

Comment (jhdalek55) [reply _data/data.yml:1109]:
```suggestion
      booktitle: "30th IEEE International Symposium on Software Reliability Engineering (ISSRE'19) Best Paper Award"
```

File: _data/data.yml [hunk 2]
```diff
@@ -462,13 +469,67 @@ people:
...
+      - &ge_yang
+            name: "Ge Yang"
+            anchor: ge_yang
+            internal: true
+            role: "Undergraduate student"
+            since: 2020
+            photo: "img/people/ge_yang"
+            interests: "Security and privacy"
+
+    - &henry_zhou
```
Comment (JustinCappos) [inline _data/data.yml:517]: It says there is an error on this line https://travis-ci.org/github/secure-systems-lab/ssl-site/builds/673149908

I think your indentation is off which is causing the issue.

Comment (jhdalek55) [reply _data/data.yml]: I may have diagnosed the problem. II thought I had accidentally removed Henry, so I re-added him. Lets see if this works.

**Comment by jhdalek55** [comment]
Yup. I think we are good now.

**Pull request merged**